package.json

```diff
@@ -49,9 +49,6 @@
     "lint-staged": "^13.0.2"
   },
   "resolutions": {
-    "bl": "~1.2.3",
-    "tough-cookie": "~2.3.3",
-    "npm": "~6.14.6",
```

Comment:
I wonder why we had to pin npm in the first place, it looks like without this resolution we don't even need it in our yarn lock file since it has no dependencies 🤔

Reply:
I have no idea why it was there. It was add more than 2 years ago. I tested all the build procedures and works np.

After we finish the `resolutions` audit (it will take long) doing a dependencies audit and review what we have, IMHO is a good idea.